system/logaccess/Azure/LogAnalytics/CurlClient/AzureLogAnalyticsCurlClient.cpp

```diff
@@ -947,7 +947,17 @@ unsigned AzureLogAnalyticsCurlClient::processHitsJsonResp(IPropertyTreeIterator
                 ForEach(*fields)
                 {
                     const char * fieldName = header.item(idx++);
-                    returnbuf.appendf("<%s>%s</%s>", fieldName, fields->query().queryProp("."), fieldName);
+                    StringBuffer encodedValue;
+                    StringBuffer rawValue(fields->query().queryProp("."));
```

Comment:
minor efficiency:

const char * rawValue = fields->query().queryProp(".");

it avoids cloning the string.